problem/array-intersect_en.md

```diff
@@ -1,4 +1,40 @@
+> This awesome solution is created by [@btjl](https://github.com/btjl) !
 
-There is no solution yet.
+## Goal
 
-Would you like to [contribute to the solution](https://github.com/BFEdev/BFE.dev-solutions/blob/main/problem/array-intersect_en.md)? [Contribute guideline](https://github.com/BFEdev/BFE.dev-solutions#how-to-contribute)
+Implement a function that takes the intersection of 2 arrays. Remove any duplicates values.
+
+## Observations
+
+1. arrays are not sorted, and might have duplicates.
+
+    - Sorting will help to identify duplicates, but doing so will immediately increase the time-complexity to `O(nlogn)` hence lets bear this in mind.
+
+2. you can modify the arrays
+
+    - Since arrays are reference types, we might not want to modify their values as it may lead to unintended effects outside of this function scope.
+
+3. you can return the items in any order, but without duplicates.
+
+    - `Set` is a data structure that ensures no duplicates may exist in it.
+
+## Implementation
```

Comment:
thanks for your PR. there are actually some other solutions as well, could you also mention them here?

Reply:
Sure, which other solutions would you like me to include? There are a few in the discussions tab.

I can add in the solution along with credits to the user if you'd like.